pkg/virtctl/create/vm/vm.go

```diff
@@ -139,6 +149,85 @@ var flags = []string{
 	CloudInitNetworkDataFlag,
 	InferInstancetypeFlag,
 	InferPreferenceFlag,
+	VolumeImportFlag,
+}
+
+type volumeSourceBlank struct {
```

Comment:
supernit - Feel free to ignore this but the discussion above about `sysprep` not being supported makes me think we would be better off naming these structs `dataVolumeSourceFoo` etc, thoughts?

Reply:
I agree that it would be better to change the name as you suggest.